source/adminguide/virtual_machines.rst

```diff
@@ -165,6 +165,10 @@ To create a VM from a template:
    functions such as mounting additional volumes and ISO images,
    live migration, and graceful shutdown.
 
+   **VMware**
+
+   When rootDiskController and dataDiskController are not specified for an instance using instance details and they are set to use osdefault in the template or the global configuration then CloudStack tries to find the recommended disk controllers for it using guest OS from the hypervisor. In some specific cases, it may create issues with the instance deployment or start operation. To overcome this, a specific disk controller can be specified at the instance or template level. For an existing instance its settings can be updated while it is in stopped state by admin.
```

Comment:
```suggestion
   When rootDiskController and dataDiskController are not specified for an instance using instance details and
   they are set to use osdefault in the template or the global configuration than CloudStack tries to find the
   recommended disk controllers for it using guest OS from the hypervisor. In some specific cases, it may create
   issues with the instance deployment or start operation. To overcome this, a specific disk controller can be
   specified at the instance or template level. For an existing instance its settings can be updated while it is in
   stopped state by admin.
```

Reply:
this is really two suggestions @shwstppr 
1. 'then' -> 'than'
2. refromat to break the line
see what you think of it.

text looks good to me, otherwise.

Reply:
forget point 1, i did some more reformulation of that first sentence.